src/backends/sentry_backend_breakpad.cpp

```diff
@@ -95,6 +95,16 @@ sentry__breakpad_backend_callback(
     SENTRY_WITH_OPTIONS (options) {
         sentry__write_crash_marker(options);
 
+#ifdef SENTRY_PLATFORM_WINDOWS
```

Comment:
it's just a closure data. uctx is not passed, but I think it still can be obtained using `getcontext()`

Reply:
Yeah I got it, but this `context` parameter is used as a closure since you pass it to `ExceptionHandler` constructor. So it has nothing to do with ucontext

I had an impression that breakpad's `getcontext` retrieves the original context, but it seems that it's just a fallback for envs that don't have `getcontext...

Reply:
Oh sorry, that was my misunderstanding of "closure data/context" then. Too bad :-(

Reply:
We should still call the `on_crash` hook for all platforms. We should probably pass `NULL` is the `uctx` parameter for the platforms where we have access to the crash context, and clearly document this.